ansible/files/envoy_config/lds.yaml

```diff
@@ -0,0 +1,334 @@
+resources:
```

Comment:
This is just temporary for AMI testing purposes. `lds.yaml` needs to be generated, the rest are static but the config generation code for envoy.yaml and cds.yaml is in worker to keep the source of truth for the YAML files in one place and DRY.

Building and testing the AMI one more time before merging, then will move `lds.yaml` elsewhere to be all-in-one specific.

Reply:
Actually, why not include the generated files only in this repo and in the AMI, as the source of truth, instead of having duplicate versions in `init-scripts`? Only `cds.yaml` would have to be overwritten to enable TLS for admin API, which is disabled in all-in-one.

Reply:
Ends up splitting up the source of truth, especially given at least some of the files (looks like lds.yaml in this case) needs to be updated in both init-scripts and the worker at the same time.

Reply:
Per offline discussion, at least for now `lds.yaml` needs to be included in this repo so that external partners are able to use the all-in-one image as-is.

Ideally we'd consolidate all such config to be injected via remote payload, such that the init process for the AMI and Fly is more consistent. If necessary for external partners, we could publish instructions and the necessary config publicly.